test/stepDefinitions/create-caseSteps.js

```diff
@@ -229,4 +229,28 @@ defineSupportCode(function ({ Given, When, Then}) {
     await baseSteps.fillOutAndSubmitForm();
   });
 
+  Given(/^I have created a case with fixed list item$/, async function() {
+    await baseSteps.navigateToCreateCasePage();
+    await caseWizardPage.interactWithField("text");
+    await caseWizardPage.interactWithField("fixed-list", "Marriage");
+    await caseWizardPage.clickContinueButton();
+    await caseWizardPage.clickSubmitCaseButton();
+  });
+
+  When(/^The fixed list item is hidden on the first page$/, async function() {
+    let fieldPresent = await caseWizardPage.isFieldPresent('fixed-list')
+    expect(fieldPresent).to.be.false;
+  });
+
+  When(/^I continue to the 3rd page$/, async function() {
+    await caseWizardPage.clickContinueButton();
+    await caseWizardPage.clickContinueButton();
+  });
+
+  Then(/^The page that satisfies show condition of the fixed list item value is displayed$/, async function() {
```

Comment:
there is a mismatch between what this method is doing and its step description. What the implementation does is checking the page header matches an expected value. 
We should make the step description more generic and aligned to the implementation. If we do that then we would also improve reusability
Also to improve the step reusability, we should avoid hardcoding values ('Conditional Page 3 - Approve a case')